src/Engine/sprite_renderer.py

```diff
@@ -2,13 +2,24 @@
 
 
 # Main game drawing function
-def draw_sprites(render_group, win, background, text_timer_box):
-    """Draws the frame to be rendered"""
-    win.fill((255, 255, 255))
+def draw_sprites(render_group, win, background):
+    """
+    Draws the frame to be rendered.
+
+    - :param render_group:
+        A pygame LayeredUpdates sprite group which contains all of the sprites to be rendered on the given frame
+    - :param win:
+        A pygame Surface object on which the sprites will be rendered
+    - :param background:
+        An image containing the game background. Must already be scaled correctly.
+    - :return:
+        None
+    """
     win.blit(background, (0, 0))
 
-    #Write over previous number
-    win.blit(text_timer_box, (10, 825))
+    # for line in player_lines:
```

Comment:
Are these needed comments? What purpose did they serve exactly before they were commented?

Reply:
It's code to render a list of lines. I was using it to debug the function that finds the distance to nearby sprites for the AI. Basically displaying the line to the sprite that was found so that I could see if it was correct. Strictly speaking we don't really need it anymore, and there's a performance cost to iterating over an empty list, but I didn't want to delete it it in case we ended up needing to do more debugging in the future. I probably should have labelled it though lol